config/locales/en.yml

```diff
@@ -29,7 +29,7 @@ en:
     show:
       link: "New Account"
       new_account: "Click New Account to sign up"
-      welcome: "Welcome to Budget App"    
+      welcome: "Welcome to Budget App"
```

Comment:
Should this be at a more generic level?

Reply:
Probably, I will need to revisit the idea of separating a home page from a landing page once I have everything setup